content/docs/howto/nodejs.md

```diff
@@ -246,6 +246,33 @@ you could then set the following at build time.
 $BP_NODE_PROJECT_PATH=node-app
 ```
 
+## Specifying Scripts to Run During Build Phase
+To specify a script or series of scripts to run during build phase, please use the 
+`BP_NODE_RUN_SCRIPTS` environment variable at build time either directly or through a 
+[`project.toml`](https://buildpacks.io/docs/app-developer-guide/using-project-descriptor). This 
+could be useful if your app uses a framework like Angular, React, or Vue where you need to run 
+scripts to build your app into a production state.
+
+For example, if your project's `package.json` has the following scripts:
+```json
+{
+  "scripts": {
```

Comment:
We might consider using examples from React instead of Angular since React is quite a bit more popular.

```
"scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "lint": "eslint src/**/*.js src/**/*.jsx",
  }
  ```
  
But this is a bit of my bias coming through for React so ignore if you feel Angular is a better example.

Reply:
As an equally biased react user, I totally agree! addressed [here](https://github.com/paketo-buildpacks/paketo-website/pull/237/commits/d3cc7f47d4e4e104aa9a0ce05ca829b61fa112c2) : )